core/chaincode/shim/chaincode.go

```diff
@@ -252,39 +264,39 @@ func (stub *ChaincodeStub) init(uuid string, secContext *pb.ChaincodeSecurityCon
 
 // ------------- Call Chaincode functions ---------------
 
-// InvokeChaincode function can be invoked by a chaincode to execute another chaincode.
+// InvokeChaincode locally calls the specified chaincode `Invoke`.
```

Comment:
People always ask me if this creates a second transaction. Should we state that it does not in the comment? Maybe that is what you were trying to imply by locally?